llap/test_llap.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from absl.testing import absltest
+from absl.testing import parameterized
+
+from integration_tests.dataproc_test_case import DataprocTestCase
+
+
+class LLAPTestCase(DataprocTestCase):
+    COMPONENT = 'llap'
+    INIT_ACTIONS = ['llap/llap.sh']
+    TEST_SCRIPT_FILE_NAME = 'run_hive_commands.py'
+
+    def verify_instance(self, name):
+        self.upload_test_file(
+            os.path.join(
+                os.path.dirname(os.path.abspath(__file__)),
+                self.TEST_SCRIPT_FILE_NAME), name)
+        self.__run_test_script(name)
+        self.remove_test_script(self.TEST_SCRIPT_FILE_NAME, name)
+
+    def __run_test_script(self, name):
+        self.assert_instance_command(
+            name, "python {}".format(self.TEST_SCRIPT_FILE_NAME))
+
+    @parameterized.parameters(
+        ("HA", ["m-0", "m-1", "m-2"]), )
```

Comment:
No test for single node clusters and normal clusters? Or are they not supported?

Reply:
no support for single clusters.